src/xdp-utils.c

```diff
@@ -1625,7 +1657,22 @@ xdg_app_info_ensure_pidns (XdpAppInfo  *app_info,
   if (app_info->u.flatpak.pidns_id != 0)
     return TRUE;
 
-  pid = xdp_app_info_get_child_pid (app_info, error);
+  root = xdg_app_info_load_brwap_info (app_info, error);
+  if (root == NULL)
+    return FALSE;
+
+  /* newer versions of bubblewrap contain the namespace
+   * information directly, so we don' thave to go via the
+   * child-pid; if this fails, we fallback to the old way */
```

Comment:
Pet peeve of mine: I *really* prefer the closing */ on the next line, with the * lining up to the rest